suite/suite_test.go

```diff
@@ -259,10 +259,14 @@ func (suite *SuiteTester) TestSubtest() {
 
 func (suite *SuiteTester) TearDownSubTest() {
 	suite.TearDownSubTestRunCount++
+	// We should get the *testing.T for the test that is to be torn down
```

Comment:
Move this comment as 3rd argument of Contains below.

Reply:
done